Compiler warning (unused param kafka_address in Fourier.hpp)

This is a minor issue, but I noticed the following compiler warning when building the software in the _dqm_ package (latest code on the _develop_ branch as of 29-Jul).

Building CXX object dqm/CMakeFiles/dqm_DQMProcessor_duneDAQModule.dir/plugins/DQMProcessor.cpp.o
In file included from /nfs/home/np04daq/.biery/dunedaq/29Jul/sourcecode/dqm/plugins/DQMProcessor.cpp:16:
/nfs/home/np04daq/.biery/dunedaq/29Jul/sourcecode/dqm/src/Fourier.hpp: In member function ‘virtual void dunedaq::dqm::FourierLink::run(dunedaq::dataformats::TriggerRecord&, dunedaq::dqm::RunningMode, std::__cxx11::string)’:
/nfs/home/np04daq/.biery/dunedaq/29Jul/sourcecode/dqm/src/Fourier.hpp:277:84: warning: unused parameter ‘kafka_address’ [-Wunused-parameter]
 FourierLink::run(dunedaq::dataformats::TriggerRecord& tr, RunningMode, std::string kafka_address)
                                                                        ~~~~~~~~~~~~^~~~~~~~~~~~~

--- a/src/Fourier.hpp
+++ b/src/Fourier.hpp
@@ -274,7 +274,7 @@ FourierLink::FourierLink(std::string name, int start, int end, int npoints)
 }
 
 void
-FourierLink::run(dunedaq::dataformats::TriggerRecord& tr, RunningMode, std::string kafka_address)
+FourierLink::run(dunedaq::dataformats::TriggerRecord& tr, RunningMode, std::string)
 {
   m_run_mark = true;
   dunedaq::dqm::Decoder dec;

